axelor-base/src/main/resources/views/Selects.xml

```diff
@@ -752,4 +752,8 @@
     <option value="empty" hidden="true"/> <!-- unused value, the select is completed in other modules -->
   </selection>
 
+  <selection name="base.registration.number.template.validation.method.select">
+    <option value="com.axelor.apps.base.service.RegistrationNumberValidationFRA">RegistrationNumberValidationFRA</option>
```

Comment:
RegistrationNumberValidationFRA should be "French registration number validation"